### DISCUSSION TITLE
How to query between two dates and two times?

### DISCUSSION THREAD

**Discussion by ricardo-valero** [prisma/prisma#12534] [Q&A]
Example 
```sql

select * 
from production 
where
    production.begin::date between '2017/05/15' and '2017/05/16'  
    and 
    production.begin::time between '00:15' and '15:00'

```

https://stackoverflow.com/questions/45803041/query-between-two-dates-and-two-times

https://stackoverflow.com/questions/59806336/postgresql-query-between-time-ranges-on-a-timestamp-column

**Comment by austincrim** [comment]
Hey @RicardoValero95 👋 ,

This is not possible in Prisma yet, though we are aware of the current limitations and hope to re-visit them soon, given capacity.

In the meantime, you can always use [`$queryRaw`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryraw) to get the result you want.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
